src/main/java/com/sda/microblogging/entity/DTO/follower/FollowerDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.sda.microblogging.entity.DTO.follower;
+
+import lombok.*;
+
+import java.sql.Date;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class FollowerDTO {
+    private Integer id;
+    private Integer userId;
+    private String userUsername;
+    private String userAvatar;
+    private Integer followerId;
+    private String followerUsername;
+    private String followerAvatar;
+    private Date followingDate;
+}
```

Comment:
Here i don't think we need both information. Because for the user we have on user object. Here we need just the user i follow information.

Reply:
This set of fields was discussed with @bkrmadtya : for UI most probably it would be convenient to have it like that for now